plugins/bright_plugin/.ci/ci_matrix.yaml

```diff
@@ -0,0 +1,54 @@
+---
+job: ufm-bright-plugin
+
+registry_host: harbor.mellanox.com
+registry_path: /swx-storage/ci-demo
+registry_auth: swx-storage
+
+env:
+  plugin_dir: bright_plugin
+  plugin_name: ufm-plugin-bright
+  DOCKER_CLI_EXPERIMENTAL: enabled
+
+kubernetes:
+  cloud: swx-k8s-spray
+
+volumes:
+  - {mountPath: /var/run/docker.sock, hostPath: /var/run/docker.sock}
+  - {mountPath: /auto/UFM, hostPath: /auto/UFM }
+
+
+runs_on_dockers:
+   - {file: '.ci/Dockerfile', arch: 'x86_64', name: 'plugin_worker', tag: 'latest'}
+
+
+steps:
+  - name: Build Plugin
+    containerSelector: "{name: 'plugin_worker'}"
+    run: |
+      cd plugins/$plugin_dir/build
+      bash -x ./docker_build.sh latest /
+      ls -l /
```

Comment:
pls save return value and exit on error.
at this point this step always returns 0 and never fails